script/lib/create-windows-installer.js

```diff
@@ -1,32 +1,45 @@
 'use strict'
 
 const electronInstaller = require('electron-winstaller')
-const fs = require('fs-extra')
+const fs = require('fs')
 const glob = require('glob')
 const path = require('path')
 
 const CONFIG = require('../config')
 
 module.exports = (packagedAppPath) => {
-  const archSuffix = process.arch === 'ia32' ? '' : '-' + process.arch
+  // const archSuffix = process.arch === 'ia32' ? '' : '-' + process.arch
```

Comment:
This and the other commented line below will be uncommented once atom.io changes are merged/deployed.